lib/command.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
can use strict mode for entire scripts https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode